### PR TITLE
Enable device preference config via url query string

### LINKED ISSUE #70: Allow to configure device preference
It would be good that users can run samples with different device by configuring the device preference.

https://webmachinelearning.github.io/webnn/#enumdef-mldevicepreference

For example, allow to select 'cpu', 'gpu' or 'default' for image classification sample and others.

### PR DESCRIPTION
**Pull request by Honry**
Simulate WebNN's device selection using various tf.js backends.
MLDevicePreference: ['default', 'gpu', 'cpu']
'default' or 'gpu': tfjs-backend-webgl, 'cpu': tfjs-backend-wasm

Exclude this feature in Code Editor sample as we don't
want to change the code from WebNN API examples.

Fixed #70

### CODE REVIEW

**Comment by Honry** [comment]
@huningxin, PTAL, thanks!

File: common/utils.js [hunk 1]
```diff
@@ -144,29 +144,35 @@ export function getMedianValue(array) {
       (array[array.length / 2 - 1] + array[array.length / 2]) / 2;
 }
 
-// Set tf.js backend
-export async function setPolyfillBackend(backend) {
-  if (!backend) {
-    // Use 'webgl' by default for better performance
-    backend = 'webgl';
-  }
+// Set tf.js backend via WebNN's 'MLDevicePreference' option
```
Comment (huningxin) [inline common/utils.js:147]: via -> based

**Review by huningxin — APPROVED** [review]
Looks good with one nit.

**Comment by Honry** [comment]
@huningxin, thanks for your reivew! Fixed the nit, let's merge it. :)

**Pull request merged**